### PR TITLE
[FEATURE] Explain why the ingress column is empty on som AWS clusters

### PR DESCRIPTION
**Pull request by oponder**
This adds an explanation for the empty ingress column on `> 10.x.x AWS`  clusters

Screenshot 2020-02-05 at 3 47 31 PM

### CODE REVIEW

File: src/components/Cluster/ClusterDetail/ClusterApps.js [hunk 1]
```diff
@@ -309,22 +312,63 @@ class ClusterApps extends React.Component {
...
+                  {this.preinstalledApps().management.map(app => {
+                    return (
+                      <div className='cluster-apps--app' key={app.name}>
+                        <img alt={`${app.title} icon`} src={app.logoUrl} />
+                        {app.name}
+                        <small>{app.version}&nbsp;</small>
+                      </div>
+                    );
+                  })}
+                </div>
```
Comment (axbarsan) [inline src/components/Cluster/ClusterDetail/ClusterApps.js]: These 2 are really similar. Do you mind creating a separate component for them?

File: src/components/Cluster/ClusterDetail/ClusterApps.js [hunk 2]
```diff
@@ -309,22 +312,63 @@ class ClusterApps extends React.Component {
...
+                        <br />
+                        <br />
+                        Read more in our{' '}
+                        <a href='https://docs.giantswarm.io/guides/installing-optional-ingress-controller/'>
+                          installing an ingress controller guide.
+                        </a>
+                      </p>
+                    </OptionalIngressNotice>
+                  ) : (
+                    undefined
```
Comment (axbarsan) [inline src/components/Cluster/ClusterDetail/ClusterApps.js]: No need to return `undefined` here, you can just write
```js
{this.props.hasOptionalIngress && (
  <Component>
  </Component
)}
```

File: src/components/Cluster/ClusterDetail/ClusterApps.js [hunk 3]
```diff
@@ -309,22 +312,63 @@ class ClusterApps extends React.Component {
...
+                  {this.props.hasOptionalIngress ? (
+                    <OptionalIngressNotice>
+                      <p>
+                        The ingress controller is optional on this cluster.
+                        <br />
+                        You can install one using our app catalog.
+                        <br />
+                        <br />
+                        Read more in our{' '}
+                        <a href='https://docs.giantswarm.io/guides/installing-optional-ingress-controller/'>
```
Comment (axbarsan) [inline src/components/Cluster/ClusterDetail/ClusterApps.js]:
```suggestion
                        <a target='_blank' href='https://docs.giantswarm.io/guides/installing-optional-ingress-controller/'>
```

File: src/components/Cluster/ClusterDetail/ClusterApps.js (hunk 1 above)
Comment (axbarsan) [reply src/components/Cluster/ClusterDetail/ClusterApps.js]: The ingress column is similar, as well

Comment (oponder) [reply src/components/Cluster/ClusterDetail/ClusterApps.js]: ok! will fix that

**Pull request merged**